protocol/protobuf/communities.proto

```diff
@@ -10,7 +10,6 @@ package protobuf;
 message Grant {
```

Comment:
Shouldn't it also be extended with expiry date as discussed offline?

Reply:
in this PR i want to just remove unneeded grant-related code for https://github.com/status-im/status-go/issues/4180, in the next one i will add logic for proof of membership